exchange_calendars/exchange_calendar_xdub.py

```diff
@@ -98,7 +98,7 @@
 
 # Ad hoc closes.
 March1BadWeather = Timestamp("2018-03-01", tz=UTC)
```

Comment:
Does this one also need UTC removed?

Reply:
Thanks for getting another review in.

I left `March1BadWeather ` as UTC because it feeds through to `special_closes_adhoc`, not `adhoc_holidays`. I've added a comment to 'XDUB' to note the difference.

A side note: the suggested fix creates a DatetimeIndex of `adhoc_holidays`, thereby requiring them to be defined consistently in terms of either UTC or tz-naive (at least within any specific calendar). At the risk of returning to the #42 UTC / tz-naive discussion 😀, I went with tz-naive in light of what they represent. Having worked through the implementation I was happy they could be defined in tz-naive terms (which was pretty clear anyway from the fact that some already were). Also, it didn't seem much point asking someone defining adhoc_holidays to set the timezone to UTC when that extra context isn't used (infact, as in some calendars, they could all just be defined as strings, e.g. "2021-07-09").

I didn't need to work through the workings of `special_closes_adhoc` and didn't want to risk introducing a bug by changing something for the sake of changing it - allowing `special_closes_adhoc` to be defined as tz-aware seemed reasonable in any case.